config/application.yml.default

```diff
@@ -407,6 +407,7 @@ reject_id_token_hint_in_logout: false
 remember_device_expiration_hours_aal_1: 720
 remember_device_expiration_minutes_aal_2: 0
 report_timeout: 0
+reproof_forcing_provider: ''
```

Comment:
```suggestion
reproof_forcing_service_provider: ''
```

Reply:
I agree that explicit is better than implicit. I patterned this after `allowed_attempts_providers` and `allowed_ialmax_providers`.

Reply:
Sorry, meant to include a comment with it. I missed that there was an existing pattern. I still lean slightly towards the more explicit version, but not terribly strongly.